src/transport/CASESession.cpp

```diff
@@ -35,11 +35,32 @@
 #include <support/CodeUtils.h>
 #include <support/ReturnMacros.h>
 #include <support/SafeInt.h>
+#include <support/BufferWriter.h>
 #include <transport/SecureSessionMgr.h>
 
 namespace chip {
 
+// Dummy x509 certificate...
+// TODO: Check whether this should be kept or not
+extern const TrustedRootIdentifier TrustedRoots[] = {
+    { 0x96, 0xDE, 0x61, 0xF1, 0xBD, 0x1C, 0x16, 0x29, 0x53, 0x1C, 0xC0, 0xCC, 0x7D, 0x3B, 0x83, 0x00, 0x40, 0xE6, 0x1A, 0x7C },
+    { 0x96, 0xDE, 0x61, 0xF1, 0xBD, 0x1C, 0x16, 0x29, 0x53, 0x1C, 0xC0, 0xCC, 0x7D, 0x3B, 0x83, 0x00, 0x40, 0xE6, 0x1A, 0x7C }
+};
+
+constexpr uint8_t kKDFSR2Info[]   = "SigmaR2";
+constexpr uint8_t kKDFSR3Info[]   = "SigmaR3";
+constexpr size_t kKDFInfoLength   = 7;
```

Comment:
Likely we can use sizeof() or strlen() to compute it, instead of hardcoding it here.
Same applies to `kKDFSEInfoLength` and `kIVLength`